docs-src/docs/reference/updateservice.md

```diff
@@ -12,7 +12,7 @@ The disadvantage of using this method is that you cannot perform A/B testing of
 **To replace a Deploy to refer to a new task definition:**
 
 1. At the command line, type the following to create a new Deploy: <br />```l0 deploy create [pathToTaskDefinition] [deployName]```<br />Note that if ```[deployName]``` already exists, this step will create a new version of that Deploy.
-2. Type the following to update the existing Service: <br />```l0 service update [existingServiceName] [deployName]```<br />By default, the Service you specify in this command will refer to the latest version of ```[deployName]```, if multiple versions of the Deploy exist.<div class="admonition note"><p class="admonition-title">Note</p><br /><p>If you want to refer to a specific version of the Deploy, type the following command instead of the one shown above: <code style="color:white;">l0 service update [serviceName] [deployName]:[deployVersion]</code></p></div>
+2. Type the following to update the existing Service: <br />```l0 service update [existingServiceName] [deployName]```<br />By default, the Service you specify in this command will refer to the latest version of ```[deployName]```, if multiple versions of the Deploy exist.<div class="admonition note"><p class="admonition-title">Note</p><br /><p>If you want to refer to a specific version of the Deploy, type the following command instead of the one shown above: <code style="color:black;">l0 service update [serviceName] [deployName]:[deployVersion]</code></p></div>
```

Comment:
Pull down my latest changes, as I reconstructed this page to not have so much html embedded in these instructions. Also clarified the syntax of the commands to run as well as linked to the CLI reference page.